docs/ref/contrib/postgres/search.txt

```diff
@@ -261,6 +266,26 @@ floats to :class:`SearchRank` as ``weights`` in the same order above::
     >>> rank = SearchRank(vector, query, weights=[0.2, 0.4, 0.6, 0.8])
     >>> Entry.objects.annotate(rank=rank).filter(rank__gte=0.3).order_by('-rank')
 
+Lexeme objects
+==============
+
+Lexeme objects allow search operators to be used with strings from an
+untrusted source. The content of each lexeme is escaped so any operators that
```

Comment:
Rewrite sounds good to my ear! If you want fine-tuning suggestions:
- "so **that** any operators that **may** exist in the string itself"

First sentence sounds fine to me too, but if you're looking for an alternative:
- "Lexeme objects allow for safely applying search operators to strings from untrusted sources."